commit 5c988de1bdaf2c4cf692e6c195375607a8c67ab0
Author: Timothy Pearson <kb9vqf@pearsoncomputing.net>
Date:   1343320500 -0500

    Fix a fatal error message
    Thanks to 'Nix' for the patch!

diff --git a/kopete/plugins/netmeeting/CMakeLists.txt b/kopete/plugins/netmeeting/CMakeLists.txt
index b6cc031..318b672 100644
--- a/kopete/plugins/netmeeting/CMakeLists.txt
+++ b/kopete/plugins/netmeeting/CMakeLists.txt
@@ -10,7 +10,7 @@
 #################################################
 
 if( NOT BUILD_KOPETE_PROTOCOL_MSN )
-  tde_message_error( "netmeeting plugin needs msn protocol.\n Add -DBUILD_KOPETE_PROTOCOL_MSN=ON to cmake flags." )
+  tde_message_fatal( "netmeeting plugin needs msn protocol.\n Add -DBUILD_KOPETE_PROTOCOL_MSN=ON to cmake flags." )
 endif( )
 
 include_directories(
